blender/arm/props_ui.py

```diff
@@ -1423,7 +1529,24 @@ def draw(self, context):
         col.prop(rpdat, 'arm_pcfsize')
         layout.separator()
 
+        def label_multiline(text: str, context: bpy.types.Context, parent):
```

Comment:
I think this function and the comments a few lines below (1546 onwards) are no longer required, right?

Reply:
Correct, removing both the duplicate method and the comments